src/Sync.js

```diff
@@ -237,56 +242,55 @@ class Sync {
     );
     return snippets;
   }
-  // getTargetFilesPaths identifies the paths to the target write files
-  async getTargetFilePaths() {
-    const writeDir = join(rootDir, this.config.target);
-    const insertPathProgress = new progress.Bar(
+  // getTargetFilesInfos identifies the paths to the target write files
+  async getTargetFilesInfos() {
+    const readTargetDirectoryProgress = new progress.Bar(
       {
-        format: fmtProgressBar("loading target file paths from " + writeDir),
+        format: fmtProgressBar("loading info for each target directory"),
         barsize: 20,
       },
       progress.Presets.shades_classic
     );
-    insertPathProgress.start(1, 0);
-    const insertFilePaths = [];
-    for await (const entry of readdirp(writeDir)) {
-      const { path } = entry;
-      insertFilePaths.push({ path });
-      insertPathProgress.setTotal(insertFilePaths.length);
-      insertPathProgress.increment();
+    readTargetDirectoryProgress.start(1, 0);
```

Comment:
It would be nice to have a comment how this progress bar is supposed to work being shared between all of the targets, it's unclear from reading the code.

Reply:
Ya, I was thinking about refactoring the progress bar situation here - 
I will address this in a subsequent PR.